src/selectors/entities/users.test.js

```diff
@@ -592,15 +592,15 @@ describe('Selectors.Users', () => {
                     general: {
                         config: {
                             TeammateNameDisplay: General.TEAMMATE_NAME_DISPLAY.SHOW_USERNAME,
-                            LockTeammateNameDisplay: 'true',
+                            LockTeammateNameDisplay: true,
                         },
                         license: {
-                            LockTeammateNameDisplay: 'true',
+                            LockTeammateNameDisplay: true,
                         },
                     },
                 },
             };
-            assert.deepEqual(Selectors.makeGetDisplayName()(newTestState, testUser1.id), 'username');
+            assert.deepEqual(Selectors.makeGetDisplayName()(newTestState, testUser1.id), 'First Last');
```

Comment:
I don't think this is the intention of the test. We should be seeing the username and not the full name of the user here. Sounds like something is wrong is the selector perhaps.

Reply:
Hmm, when changing it back to `username`, the test fails. What could be wrong here? 🤔

Reply:
Have a look at the selector and investigate a bit. Perhaps some `console.log` statements might help out?

Reply:
Taking another look at this, it's failing because you changed `license.LockTeammateNameDisplay` to a boolean in the tests, when it's still expecting a string in the other code. You'll want to revert that since we're not changing the types in the license with this.

Reply:
Thank you @hmhealey, that was exactly the reason tests were failing. Should be good to go now:)